lib/oidcstrategy.js

```diff
@@ -542,6 +573,7 @@ Strategy.prototype.authenticate = function authenticateStrategy(req, options) {
   var prompt = options && options.prompt;
   var extraAuthReqQueryParams = options && options.extraAuthReqQueryParams;
   var extraTokenReqQueryParams = options && options.extraTokenReqQueryParams;
+  var response = options && options.response;
```

Comment:
@lovemaths `options.response` is always undefined in my case.

In express 4, response can be accessed with `req.res`. Was this PR developed with this in mind ?